### PR TITLE
Added bug number to changelog to please OBS bot

### PR DESCRIPTION
**Pull request by ancorgs**

### CODE REVIEW

**Comment by jreidinger** [comment]
LGTM

File: package/yast2.changes [hunk 1]
```diff
@@ -2,6 +2,7 @@
 Tue Jan 26 09:18:44 UTC 2016 - ancor@suse.com
 
 - Added UI::InstallationDialog as an specialization of UI::Dialog
+  (part of bsc##893825)
```
Comment (mvidner) [inline package/yast2.changes]: Are you sure the double hash `##` is OK?

File: package/yast2.changes [hunk 2]
```diff
@@ -2,6 +2,7 @@
 Tue Jan 26 09:18:44 UTC 2016 - ancor@suse.com
 
 - Added UI::InstallationDialog as an specialization of UI::Dialog
```
Comment (mvidner) [inline package/yast2.changes]: I can detect you speak Spanish because you write a**n** specialization :smile:

**Comment by ancorgs** [comment]
@mvidner Added *an* special commit fixing both errors. :wink:

**Comment by mvidner** [comment]
LGTM!

**Pull request merged**